README.md

```diff
@@ -66,6 +66,21 @@ import VuePaycard from "vue-paycard";
 Vue.use(VuePaycard);
 ```
 
+For Vue 3
```

Comment:
Are you sure you tested this on Vue 3? Because there's a few things that I believe are not Vue 3 compatible, such as `beforeDestroy` lifecycle hook, and right now I don't want to drop Vue 2 support yet.